### ISSUE TITLE
not detecting changes to SQLite in WAL mode

### ISSUE THREAD

**Issue by malhal** [emcrisostomo/fswatch#150]
On OS X 10.11.6 with fswatch 1.9.2

```
$ ls test
Untitled.sqlite		Untitled.sqlite-shm	Untitled.sqlite-wal
$fswatch test
```

Make a change to the database and no file changes are reported despite them having changed, timestamps updated. Reporting this issue because the Readme said there are no known issues with FSEvents method.

**Comment by emcrisostomo** [comment]
Hi @malhal,

I've just tried to replicate it and I confirm that the FSEvents API does _not_ return an event in the situation you describe. If I'm mistaken about something, I don't know what it is. The fact that Finder doesn't detect the change either is a strong indicator that FSEvents is incapable of detecting it.

This issue is worth some research time.

**Comment by holyspidoo** [comment]
Exact same problem I ran into :) 

Any hope for this issue? Or is there a better way altogether to monitor sqlite files for changes (instead of monitoring files changes)?

**Comment by nicad** [comment]
Also ran into this while looking at sqlite files from iCloud

**Comment by hepcat72** [comment]
Yup.  +1.  I wrote a script to process reminders changes and thought using fswatch might be an efficient means of deciding when it should run...

I guess it shouldn't be too bad to use `-m poll_monitor` for my use case with a single file.

**Comment by Hadrien-lcrx** [comment]
Same issue here: I'm monitoring changes on a `.db` file.

On a Mac, running `fswatch "/Users/hadrien/Library/Group\ Containers/34YW5XSRB7.com.omnigroup.OmniFocus/com.omnigroup.OmniFocus3/com.omnigroup.OmniFocusModel/OmniFocusDatabase.db" | xargs -n 1 -I {} echo "working"` doesn't print `working` when the file is modified despite the <Modified:> field getting updated when I push a change to the database.
